README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  word_document: default
+  html_document: default
+---
```

Comment:
This is overcomplicated! This is something called a YAML header and only goes at the top of .Rmd files. It doesn't have a place here.